wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
+
```

Comment:
What do you think about adding an attribution to the repo within this script for transparency?

Reply:
good point. I noticed the project is listed on packagist, I'm going to see if I can include it that way instead so attribution is baked in, otherwise will add something here.

Reply:
Found this open PR on that repo, figured I'd use that: https://github.com/vishnubob/wait-for-it/pull/95